templates/services/trainingdata-deployment.yml

```diff
@@ -29,7 +29,7 @@ spec:
 {{ end }}
       containers:
       - name: ffdl-trainingdata-container
-        image: registry.ng.bluemix.net/{{.Values.docker.namespace}}/{{.Values.docker.image_prefix}}metrics:{{.Values.trainingdata.version}}
+        image: docker.io/{{.Values.docker.namespace}}/{{.Values.docker.image_prefix}}metrics:{{.Values.trainingdata.version}}
```

Comment:
How come this is not using `{{.Values.docker.registry}}`?

Reply:
Thanks, good catch! Will fix in a follow-up PR.